notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,340 @@
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit ('.venv')",
+   "metadata": {
+    "interpreter": {
+     "hash": "74f679a4fc854d75b381693882d369b7543a476b4b9cfda76eebb965e9131c3c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install -q timesketch_api_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install ../api_client/python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timesketch_api_client import search\n",
+    "from picatrix.lib import state as state_lib\n",
+    "\n",
+    "import altair as alt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from timesketch_api_client import config\n",
+    "from timesketch_import_client import helper\n",
+    "from timesketch_import_client import importer\n",
+    "from timesketch_api_client import search\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "source": [
+    "# Create the sketch and import some data"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client = config.get_client(confirm_choices=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client.create_sketch(name=\"test\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sketch = ts_client.get_sketch(1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import_helper = helper.ImportHelper() \n",
+    "\n",
+    "with importer.ImportStreamer() as streamer:\n",
+    "  streamer.set_sketch(sketch)\n",
+    "  streamer.set_config_helper(import_helper) \n",
+    "\n",
+    "  streamer.set_timeline_name('sigma_events.jsonl')\n",
+    "\n",
+    "  streamer.add_file('../test_tools/test_events/sigma_events.jsonl')"
+   ]
+  },
+  {
+   "source": [
+    "# Multiple Rules\n"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rules = ts_client.list_sigma_rules()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for rule in rules:\n",
+    "    rule.__dict__"
```

Comment:
do we want to expose the `__dict__` here? Rather not just have a `print(rule.X)` or `print(f'{rule.X} something {rule.Y}')` instead of the dict....

Reply:
will remove the cell, the idea what is in the thing is in the next cell already